### ISSUE TITLE
Incorrect prefix on API

### ISSUE THREAD

**Issue by svkovalenko** [igaster/laravel_cities#36]
I use Laravel 7, Api routes added automatically. I don't need to add \Igaster\LaravelCities\Geo::ApiRoutes(); in api.php.
After add \Igaster\LaravelCities\Geo::ApiRoutes();. in group routes with prefix 'api'. This prefix don't appears.
But in routes.php use prefix 'geo'. 
Please changed prefix to 'api/geo/' in routes.php to use correct path 'api/geo/item/{id}'.
Thank you.

**Comment by abbaseya** [comment]
I agree with @SergiyKovalenko this little change in the latest release wasted far too long just to figure out that the registered routes actually changed since Jan 13 – commit [bc39916](https://github.com/igaster/laravel_cities/commit/bc39916fd310a16a52e7b8e11ac2c4d765a0ff3a#diff-afa1b594e46cb5864abbeb10131850ef).
Either updating your readme to reflect the new routes prefix `/geo` or just restore those back to `/api/geo` please.
